internal/installer/elasticagent_tar_macos.go

```diff
@@ -81,7 +81,7 @@ func (i *elasticAgentTARDarwinPackage) Exec(ctx context.Context, args []string)
 
 // Enroll will enroll the agent into fleet
 func (i *elasticAgentTARDarwinPackage) Enroll(ctx context.Context, token string, extraFlags string) error {
-	cmds := []string{"/elastic-agent/elastic-agent", "install"}
+	cmds := []string{"./elastic-agent/elastic-agent", "install"}
```

Comment:
```suggestion
	cmds := []string{"sudo", "./elastic-agent/elastic-agent", "install"}
```

Reply:
We probably need to add sudo to all operations, such as uninstall

Reply:
Thanks Manu. I just added https://github.com/elastic/e2e-testing/pull/2626/commits/57888163234f9700224a092c6d53d328acd905e6 and also https://github.com/elastic/e2e-testing/pull/2626/commits/7a9171a6a3f310fde5b6969e11a8cc2e2b4aa6da

🤞